src/Sentry/Exceptions/SentryStackFrame.cs

```diff
@@ -134,6 +134,14 @@ public sealed class SentryStackFrame : IJsonSerializable
         /// </remarks>
         public long? InstructionOffset { get; set; }
 
+        /// <summary>
+        /// Optionally changes the addressing mode. The default value is the same as
+        /// `"abs"` which means absolute referencing. This can also be set to
+        /// `"rel:DEBUG_ID"` or `"rel:IMAGE_INDEX"` to make addresses relative to an
+        /// object referenced by debug id or index.
+        /// </summary>
+        public string? AddressMode { get; set; }
```

Comment:
seems there is no real usage of AddressMode. should the be a test that uses it?

Reply:
test added